CIP-0010/registry.json

```
@@ -154,5 +154,9 @@
   {
     "transaction_metadatum_label": 61286,
     "description": "CIP-0015 - Catalyst deregistration"
-  }
+  },
+  {
+    "transaction_metadatum_label": "????",
+    "description": "CIP-???? - KERI-backed metadata attestations"
```

```suggestion
    "transaction_metadatum_label": "170",
    "description": "CIP-0170 - KERI-backed metadata attestations"
```
This change is just a guideline because it (also) needs to be collated back in numerical sequence earlier in the array... which you're welcome to leave comma-terminated 🙏

---

p.s. also fixed the inapproriate quotes I left in (causing the familiar validation failure) and moved back into numerical sequence.  Sorry to manhandle your branch but wanted to fix the pending error email from our GitHub CI & fix your problem conclusively.